client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.controller.js

```diff
@@ -62,15 +82,15 @@ class CloudProjectComputeInfrastructureOpenstackClientCtrl {
         this.minimized = !this.minimized;
         this.maximized = false;
         this.savePrefs();
-        $event && $event.stopPropagation();
+        $event.stopPropagation();
     }
```

Comment:
Why should we remove the 'undefined' check?

Reply:
`minimize` and `maximize` methods always has `$event` as argument.